oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/FullGCStatsCollectorImpl.java

```diff
@@ -83,9 +84,15 @@ class FullGCStatsCollectorImpl implements FullGCStatsCollector {
 
     private final CounterStats counter;
     private final CounterStats failureCounter;
+    private static String METRICS_QUALIFIED_NAME_PREFIX;
 
     FullGCStatsCollectorImpl(StatisticsProvider provider) {
+        this(provider, false);
+    }
+
+    FullGCStatsCollectorImpl(StatisticsProvider provider, boolean pushMetrics) {
         this.provider = provider;
+        this.METRICS_QUALIFIED_NAME_PREFIX = pushMetrics ? FULL_GC_PUSH_METRICS_PREFIX : FULL_GC;
```

Comment:
I would change the variable name to indicate that they are from OAK-RUN.

`PushMetrics` doesn't sound very intuitive to me.